python/lsst/sims/photUtils/Sed.py

```diff
@@ -106,6 +106,10 @@ def __init__(self, wavelen=None, flambda=None, fnu=None, badval=numpy.NaN, name=
         self.zp = -2.5*numpy.log10(3631)
         self.name = name
         self.badval = badval
+
+        self._physParams = PhysicalParameters()
+        self._seedingDefaults = LSSTdefaults()
```

Comment:
I'm actually going to change this and make _seeingDefaults a dict copied straight from LSSTdefaults, i.e.

`self._seeingDefaults = LSSTdefault()._seeing`